scripts/JsonRpc.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+
+class JsonRpc(requests.Session):
+    rpc_path = '/json-rpc'
+
+    def __init__(self, address):
+        super(JsonRpc, self).__init__()
+        self.host = address + self.rpc_path
+        self._id = 0
+
+    def request(self, http_method, url, json=None, **kwargs):
```

Comment:
I am overriding the [Session.request](https://github.com/requests/requests/blob/c6a634715637e971e485766bfa37c73613ab7b82/requests/sessions.py#L435) of the requests API, since I wasn't sure which HTTP method we will be using.
All convenience methods (`post`, `put`, `delete`, …) will call `Session.request(self, method, url, **kwargs)` with the `method` set to `POST`, `PUT`, `DELETE`, … respectively.